src/cogs/unquote_link.py

```diff
@@ -0,0 +1,36 @@
+import re
+import urllib.parse
+
+from discord.ext import commands
+
+URL_PATTERN = re.compile(r"http(s)?://([-\w]+\.)+[-\w]+(/[-\w./?%&=~]*)?")
```

Comment:
```suggestion
URL_PATTERN = re.compile(r"https?://([-\w]+\.)+[-\w]+[-\w./?%&=~]*")
```
`s` の後ろにそのまま `?` を付ければ、`?` の範囲は1文字なので、括弧で囲む必要がない

ドメイン以降の文字は、`*`で0文字以上にしてるんだから、括弧と `?` は要らないと思った
こちらは可読性が少し下がる？

Reply:
建前：グルーピングしてあればいざって時取り出せて便利かもなって思った
本音：拾ってきたのめんどいからそのまま使った

訂正後見てみたらこっちの方がスッキリしてていいね。